src/ColorPicker.webmodeler.ts

```diff
@@ -0,0 +1,76 @@
+import { Component, createElement } from "react";
+
+import { Alert } from "./components/Alert";
+import { Input } from "./components/Input";
+import { Button } from "./components/Button";
+import { Label } from "./components/Label";
+
+import { ColorPicker } from "./components/ColorPicker";
+
+import ColorPickerContainer, { ColorPickerContainerProps } from "./components/ColorPickerContainer";
+
+declare function require(name: string): string;
+
+type VisibilityMap = {
+    [P in keyof ColorPickerContainerProps]: boolean;
+};
+
+// tslint:disable-next-line class-name
+export class preview extends Component<ColorPickerContainerProps, {}> {
+    render() {
+        const maxLabelWidth = 11;
+        return createElement("div", {},
+            createElement(Alert, { className: "widget-charts-color-picker-alert" },
+                ColorPickerContainer.validateProps(this.props)
+            ),
+            this.props.label.trim()
+                ? createElement(Label, {
```

Comment:
Not neat, using ternary operator for a multi-line component... extract

Reply:
Done